sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/UnusedPrivateMemberTest.cs

```diff
@@ -686,13 +686,13 @@ private class Class1 { }
 
         [TestMethod]
         [TestCategory("Rule")]
-        public void Assembly_Level_Attributes_FalsePositive()
+        public void Assembly_Level_Attributes()
         {
             Verifier.VerifyCSharpAnalyzer(@"
 [assembly: System.Reflection.AssemblyCompany(Foo.Constants.AppCompany)]
 public static class Foo
 {
-    internal static class Constants // Noncompliant, False Positive; we cannot detect usages from assembly level attributes.
+    internal static class Constants // Compliant, detect usages from assembly level attributes.
```

Comment:
We can now detect internal members used in assembly-level attributes because in the second pass we analyze all whole syntax trees in the compilation, not just the class declarations